db/migrate/20240124214427_set_default_values_to_zero_in_observation_accuracy_experiments.rb

```diff
@@ -0,0 +1,8 @@
+# frozen_string_literal: true
+
+class SetDefaultValuesToZeroInObservationAccuracyExperiments < ActiveRecord::Migration[6.1]
+  def change
+    change_column_default( :observation_accuracy_experiments, :responding_validators, 0 )
```

Comment:
I tested reverting this migration and got an error `change_column_default is only reversible if given a :from and :to option.`. Ideally this would list the previous value so this could be rolled back without an error, e.g. 
```
change_column_default( :observation_accuracy_experiments, :validated_observations, from: nil, to: 0 )
```
I think its a good general practice to test rolling back new migrations before committing so be sure they can be created and destroyed, in case we ever need to roll back a deploy. That can be tested with `rake db:migrate:redo` or `rake db:migrate:down VERSION=20240124214427`